### PR TITLE
language: added vhs

### PR DESCRIPTION
**Pull request by rsteube**
Added [vhs](https://github.com/charmbracelet/vhs)

### CODE REVIEW

File: runtime/queries/vhs/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "LetterSpacing"
+  "TypingSpeed"
+  "LineHeight"
+  "Padding"
+  "Theme"
+  "Width" ] @type
+
+[ "@" ] @operator
+(control) @function.macro
+(float) @float
```
Comment (the-mikedavis) [inline runtime/queries/vhs/highlights.scm:31]:
```suggestion
(float) @constant.numeric.float
```

File: runtime/queries/vhs/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+(integer) @number
```
Comment (the-mikedavis) [inline runtime/queries/vhs/highlights.scm:32]:
```suggestion
(integer) @constant.numeric.integer
```

File: runtime/queries/vhs/highlights.scm [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+(comment) @comment @spell
```
Comment (the-mikedavis) [inline runtime/queries/vhs/highlights.scm:33]:
```suggestion
(comment) @comment
```

File: runtime/queries/vhs/highlights.scm [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+[(path) (string) (json)] @string
```
Comment (the-mikedavis) [inline runtime/queries/vhs/highlights.scm:34]:
```suggestion
(path) @string.special.path
[(string) (json)] @string
```

File: runtime/queries/vhs/highlights.scm [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+(time) @symbol
```
Comment (the-mikedavis) [inline runtime/queries/vhs/highlights.scm:35]:
```suggestion
(time) @string.special.symbol
```

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Thanks!

**Comment by maaslalani** [comment]
Thanks @rsteube @the-mikedavis!

**Pull request merged**